lib/manager/mix/artifacts.ts

```diff
@@ -47,14 +47,7 @@ export async function updateArtifacts({
 
   const cmdParts =
     config.binarySource === BinarySource.Docker
-      ? [
-          'docker',
-          'run',
-          '--rm',
-          `-v ${cwd}:${cwd}`,
-          `-w ${cwd}`,
-          'renovate/elixir mix',
-        ]
+      ? ['docker', 'run', '--rm', `-v ${cwd}:${cwd}`, `-w ${cwd}`, 'elixir mix']
```

Comment:
Doesn't look right. Is this because it was a legacy docker run that we hadn't migrated yet?

Reply:
Yes, looks like some missing refactoring here.